### ISSUE TITLE
Index card text almost invisible in dark themes.

### ISSUE THREAD

**Issue by anicede** [olivierkes/manuskript#183]
There doesn't seem to be a way to change the summary text colour of the index cards, and it's a very light grey on white in dark themes. I can change the background colour of the index cards, but the options are all very light (and I also can't figure out how to define the 'POV, Label, Progress, Compile' colours - is there something obvious I'm missing here?).

This happens with both the old, and the new index card styles.

I'm running Manuskript on Peppermint Eight with the out of the box theme, and on Xubuntu with a dark theme I can't remember the name of r/n, Application style GTK+, and it's happening on both. I can change to Fusion to see the text, but light screens are rough on the old eyes.

(loving the new nav icons and text, btw, also, thank you for your work on this bit of software, it's saving my sanity r/n)

**Comment by olivierkes** [comment]
Hi !
You're right, you cannot chose the colors. They are hardcoded in manuskript without taking into account user's theme (which is kind of stupid). You can define the colors of each specific character and label (for POV and Label), but not for progress or compile.

Can you please:

- Post one or more screenshot(s) (you can simple paste it in a comment, github is smart). Notably how it looks in GTK+, and the difference between Peppermint and Xubuntu themes.
- Tell me precisely which version of manuskript you're running and if you see that both with old and new index card style? (Cf. #158) (If you see the new nav icons, I guess you're running the latest version?)

Thanks !

**Comment by anicede** [comment]
Peppermint 8 on 32bit netbook - running from source, installed yesterday, so version 0.5.0 I believe.
![peppermint-screenshot](https://user-images.githubusercontent.com/24648031/32296139-3a0d2db4-bfb0-11e7-9ad4-093d4cff6527.png)

Xubuntu 16.04 on 64bit laptop - installed .deb a week or so ago, so version 0.4.0?
![xubuntu-screenshot](https://user-images.githubusercontent.com/24648031/32296210-7831c532-bfb0-11e7-83e9-26a945d8ca2c.png)

As you can see, it's happening in both old and new index card style. The right hand tabs are doing the same thing (though that's not a major, I don't need to be able to read them easily), as is the Navigation header, but the selected nav tab is switching to dark text.

I'll have a play with defining label colours, and see if that gets me a fix. If not, the metadata tab lets me view the summaries on a scene by scene basis, and I'll switch to Fusion when I need to see them all at once while I reoutline (I'm editing a first draft).

**Comment by olivierkes** [comment]
OK I see. The card is painted using custom colors, except for the text that uses the theme color.

If you run manuskript from source, **please try the develop branch**. The last commit should make cards usable:

![image](https://user-images.githubusercontent.com/1192933/32323254-b3f7370c-bfc7-11e7-883b-5dd4b3252d6c.png)

It's not perfect, though. For example, if setting "Tree View > Text = Compile", then it becomes unreadable:

![image](https://user-images.githubusercontent.com/1192933/32323298-e4bb1c32-bfc7-11e7-9eef-50b477379df6.png)

**Comment by anicede** [comment]
Installed develop branch on the netbook, and it's marvellous, my eyes thank you :) Will be updating the laptop first thing in the morning!

**Comment by olivierkes** [comment]
I worked on that this morning. I removed almost every harcoded colors in manuskript, using only colors from desktop theme. The result should be much smoother, whatever the theme.

I'd be happy to have some feedback (especially on windows and mac), since I cannot test there. Just to be sure that I did't break anything visually.

It's on the develop branch.

![image](https://user-images.githubusercontent.com/1192933/32784816-28d4bf7c-c950-11e7-9910-0f49f95aa6a7.png)
